docker_config/dockerfiles/Dockerfile.default-cpu

```diff
@@ -60,26 +54,11 @@ RUN wget http://scala-lang.org/files/archive/scala-$SCALA_VERSION.deb && \
     apt-get clean && \
     apt-get autoremove && \
```

Comment:
We should combine all apt-get install commands into one single command if possible. I believe this should save a bunch of space (as we'd store far less layers).